guides/Operate-production-services.md

```diff
@@ -36,24 +36,29 @@ Some redirections are done at the DNS level. See the "Web Forwarding" tab of Gan
 - `http(s)://www.openfisca.org` -> `https://openfisca.org`
 - `http(s)://doc.openfisca.fr` -> `https://openfisca.org/doc/`
 - `http(s)://www.openfisca.fr` -> `https://fr.openfisca.org`
+- `http(s)://fr.openfisca.org` -> `https://openfisca.org/fr`
 
 > `http(s)://` means that both `http://` and `https://` URLs are redirected.
 
 ## Services
 
-### Documentation
+### Website
 
-Source code available on <https://github.com/openfisca/openfisca-doc/>
+Source code available on <https://github.com/openfisca/openfisca.org/>
 
-It is deployed to <https://openfisca.org/doc/>.
+It is deployed to <https://openfisca.org/> by continuous deployment provided by CircleCI.
+
+It is hosted by GitHub Pages.
 
-It is hosted by [Netlify](https://www.netlify.com/).
+The [settings](https://github.com/openfisca/openfisca.org/settings/pages) of the GitHub repository defines a custom domain name (`openfisca.org`), and the registrar has been set up to point the `A` and `AAAA` DNS records to GitHub Pages' servers.
```

Comment:
👍 And explicit where the registrar has been set up?
```suggestion
The [settings](https://github.com/openfisca/openfisca.org/settings/pages) of the GitHub repository defines a custom domain name (`openfisca.org`), and on our current gandi.net hosting service, the registrar has been set up to point the `A` and `AAAA` DNS records to GitHub Pages' servers.
```

Reply:
Good point. I prefer to avoid repeating the information, so I'll rather use an internal link.